pkg/store/helpers.go

```diff
@@ -12,21 +12,38 @@ func createEndpoints(addrs []string, scheme string) (entries []string) {
 	return entries
 }
 
-// Formats the key
-func format(key string) string {
-	return fullpath(splitKey(key))
-}
-
-// Formats the key partially (omits the first '/')
-func partialFormat(key string) string {
-	return partialpath(splitKey(key))
+// Normalize the key for each store to the form:
+//
+//     /path/to/key
+//
+// If omit is set to 'true', this will omit
+// the first '/' character:
+//
+//     path/to/key/
+//
+func normalize(key string, omit bool) string {
```

Comment:
I'm worried that some K/V (only if we consider adding more backends in the future) may forbid using special characters like `.`, `-`, and especially `/` as the first character of a Key. Either way this is a cosmetic change so I don't mind changing it to have a private method in Consul for now :)